config/params.ini

```
@@ -14,6 +14,8 @@ hidden_transparency=0.35
 ; In case you get a warning about the LocalPref file not being found, you can either ignore the warning,
 ; or specify the correct path for your system here
 local_prefs_path=
+; Which browser to use to get builds, Chrome, Edge, or Firefox are currently supported
+browser=Edge
```

```suggestion
browser=Chrome
```
Nitpick: Chrome seems to be the most used, so set the default to it

---

Would agree, but I'd argue most people are running on Windows, so should always have Edge installed. I myself use Firefox over Chrome, but I'm never going to die on a hill defending Edge, so I'll make this change :D